Renaming bit locker and device encryption (#6244)
MicrosoftDocs/windows-itpro-docs: windows/security/information-protection/bitlocker_DeviceEncryption/bitlocker-how-to-deploy-on-windows-server.md

## <a href="" id="installing-bitlocker-"></a>Installing BitLocker

> This section will provide an overview about how to install BitLocker on Windows Server 2012 or later.

Reviewer: Is there a particular reason for choosing the future tense here? I would think that future tense belongs in a comment line.
Otherwise, if the sentence refers to existing material further below, I recommend using the present tense:
```suggestion
This section provides an overview of how to install BitLocker on Windows Server 2012 or later.
```

Author: The answer is that English is not my mother tongue. I have not consciously changed the tense. Even though I try hard to write a good English, it seems to not work in exceptional cases ;)
